rmf_fleet_adapter/schemas/event_description_PerformAction.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/open-rmf/rmf_ros2/main/rmf_fleet_adapter/schemas/event_description_PerformAction.json",
+  "title": "PerformAction event Description",
+  "description": "Have the robot perform a custom action",
+  "type": "object",
+  "properties": {
+    "category": {
+      "description": "A string that describes the nature of this action. It should match that passed into FleetUpdateHandle::add_performable_action()",
+      "type": "string"
+    },
+    "description": {
+      "description": "A dictionary describing the action to be performed.",
+      "type": "object"
+    },
+    "unix_millis_action_duration_estimate": {
+      "description": "(Optional) The estimated duration for this action",
```

Comment:
I wonder if we should change `(Optional)` to `(Recommended)`. If the planner assumes 0 time for a looong action, then the planning and dispatching could get really messed up.

Reply:
Alternatively, we could have some way for system integrators to give the fleet adapter an estimate of the time (maybe also the finish location) based on the information provided in the description.

But these are really thoughts for future enhancements, not really important right now.

Reply:
I've updated to `Recommended`.